### ISSUE TITLE
As an Industry Reporter, I want a card that goes to reporting from my dashboard so that I can view that module

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-reporting#194]
#### Description:

Blocked: #274 #1653

We need to give Operators a way to get to the reporting module from the main dashboard they get sent to after logging in. 
Step 1 - view the link to Reporting
Step 2 - Clicking the link takes the user to the Reporting Dashboard

[Wireframe here](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=7789%3A884&mode=design&t=cSGQYznXhtd4rwZd-1)

#### Acceptance Criteria:

Given that I am an external operator
And I have logged in to BCIERS
Then there is a link on the dashboard that takes me to the reporting module 

Given that I am an external operator
And I have logged in to BCIERS
And I click on the link to the reporting module
Then I can see the landing page (also called the "Reporting Dashboard") of the reporting module

#### Development Checklist:

- [ ] Added card in the registration landing page
- [ ] ~~Card should be behind a feature flag (growthbook.io)~~ N/A: Not needed now that BCIERS is split out. 
- [ ] Card links to a reporting specific URL (the one specified by the helm chart)
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [x] User story is included
- [x] User role and type are identified
- [x] Acceptance criteria are included
- [x] Wireframes are included (if required)
- [x] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [x] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements (TBC)
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Review with UX Designer
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By

 #274 #1653

#### Notes:

-

**Comment by patriciarussellCAS** [comment]
Added the Backlog Refinement label again so we can have a more in-depth conversation about acceptance criteria and the Dev checklist. 
For example, this may be blocked by the fact that this is our first ticket to touch Moose code; we need to make sure we have guidelines confirmed about how we go about this before doing any Dev work.

**Comment by patriciarussellCAS** [comment]
To confirm - does clicking the link take you immediately to the current reporting year dashboard?

**Comment by pbastia** [comment]
This has likely already been implemented as part of the common dashboard, we just need to confirm that the functionality works as expected

**Comment by patriciarussellCAS** [comment]
@suhafa design review

**Comment by joshgamache** [comment]
@suhafa @patriciarussellCAS 
Address for this on (when logged in) is `/dashboard`: [https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/dashboard](https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/dashboard).
The "Reporting" card now has an active link for _Submit Annual Report(s)_ which takes you to the operations page.

**Comment by patriciarussellCAS** [comment]
Thanks @joshgamache - I was able to go from the main dashboard to a second page with "submit annual report" and "view past submissions" - though this wasn't the intended action. My understand  is that when you click on the Reporting card on the dashboard, this would take you directly to the "Reporting Dashboard" page. 

Also, on the second page, when I click on "submit annual reports" I get sent to a blank page with an error. 
"Application error: a client-side exception has occurred (see the browser console for more information)."
@suhafa tagged you for visibility.

**Comment by patriciarussellCAS** [comment]
This has been reviewed and approved. Just noting that eventually the intended flow would be to go straight to the reporting dashboard form the "reporting" card on the main dashboard.

**Comment by patriciarussellCAS** [comment]
@dleard to create a follow-up ticket to review URL patterns and breadcrumbs to ensure they align with our planned paths. [See ticket #268]

For this specific ticket, the intended path is to go from the main dashboard to the reporting dashboard. 
The current path is Dashboard > Reporting > Operations 
We shouldn't have this middle step: https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/reporting
Tagging @joshgamache for visibility
